components/script_traits/lib.rs

```diff
@@ -492,21 +492,15 @@ pub enum IFrameSandboxState {
     IFrameUnsandboxed
 }
 
-/// Specifies the information required to load a URL in an iframe.
+/// Specifies the information required to load an iframe.
 #[derive(Deserialize, Serialize)]
 pub struct IFrameLoadInfo {
-    /// Load data containing the url to load
-    pub load_data: Option<LoadData>,
     /// Pipeline ID of the parent of this iframe
     pub parent_pipeline_id: PipelineId,
     /// The ID for this iframe.
     pub frame_id: FrameId,
-    /// The old pipeline ID for this iframe, if a page was previously loaded.
-    pub old_pipeline_id: Option<PipelineId>,
     /// The new pipeline ID that the iframe has generated.
     pub new_pipeline_id: PipelineId,
-    /// Sandbox type of this iframe
-    pub sandbox: IFrameSandboxState,
```

Comment:
Why do we not need the sandbox state for the frame when it's initially created? (Really the sandbox state should be a property of the frame rather than of the pipelines.)